change/@fluentui-react-806ea81d-5ece-4e46-bab4-7864fe8ce896.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv8 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32264&commitId=8d59a67b916c716308f70de4078f4bcfb1bb51eb&env=prod&clientType=FLUENTUI)


  <details><summary><b>Checkbox</b> 13 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Checkbox.Checked disabled.chromium.png|299| Changed|
|Checkbox.Controlled Indeterminate.chromium.png|299| Changed|
|Checkbox.Custom render Checkbox.chromium.png|299| Changed|
|Checkbox.Checked.chromium.png|299| Changed|
|Checkbox.End RTL.chromium.png|306| Changed|
|Checkbox.Unchecked RTL.chromium.png|306| Changed|
|Checkbox.Controlled Indeterminate disabled.chromium.png|299| Changed|
|Checkbox.Unchecked disabled.chromium.png|299| Changed|
|Checkbox.Multi-line Checkbox.chromium.png|299| Changed|
|Checkbox.End.chromium.png|299| Changed|
|Checkbox.Unchecked.chromium.png|299| Changed|
|Checkbox.Uncontrolled Indeterminate disabled.chromium.png|299| Changed|
|Checkbox.Uncontrolled Indeterminate.chromium.png|299| Changed|

</details>


  <details><summary><b>Checkbox Indeterminate</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Checkbox Indeterminate.Uncontrolled Indeterminate defaultChecked checkbox.chromium.png|271| Changed|
|Checkbox Indeterminate.Uncontrolled Indeterminate checkbox.chromium.png|271| Changed|
|Checkbox Indeterminate.Controlled Indeterminate checkbox without onChange callback.chromium.png|271| Changed|
|Checkbox Indeterminate.Uncontrolled and Controlled Indeterminate checkbox.chromium.png|271| Changed|

</details>


  <details><summary><b>ChoiceGroup</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ChoiceGroup.Root.chromium.png|317| Changed|
|ChoiceGroup.Required.chromium.png|317| Changed|
|ChoiceGroup.With default size images.chromium.png|317| Changed|
|ChoiceGroup.With Icons RTL.chromium.png|316| Changed|
|ChoiceGroup.With icons.chromium.png|317| Changed|
|ChoiceGroup.With large size images.chromium.png|317| Changed|

</details>


  <details><summary><b>DocumentCard</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|DocumentCard.Compact with preview image.chromium.png|357| Changed|
|DocumentCard.Compact with preview list.chromium.png|357| Changed|
|DocumentCard.Not truncated.chromium.png|357| Changed|
|DocumentCard.With secondary title style.chromium.png|357| Changed|

</details>


  <details><summary><b>FolderCover</b> 5 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|FolderCover.Large Media Cover.chromium.png|312| Changed|
|FolderCover.Large Default Cover.chromium.png|312| Changed|
|FolderCover.Shared Cover.chromium.png|312| Changed|
|FolderCover.Small Default Cover.chromium.png|312| Changed|
|FolderCover.Small Media Cover.chromium.png|312| Changed|

</details>


  <details><summary><b>Icon</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Icon.Root.chromium.png|303| Changed|
|Icon.Image.chromium.png|303| Changed|
|Icon.Color.chromium.png|303| Changed|

</details>


  <details><summary><b>MediaTile</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|MediaTile.Media tile with nameplate hidden until hover.chromium.png|312| Changed|
|MediaTile.Media tile with single activity line.chromium.png|312| Changed|
|MediaTile.Media tile with very long name and activity.chromium.png|312| Changed|
|MediaTile.Media tile with two activity lines.chromium.png|312| Changed|

</details>


  <details><summary><b>Signals</b> 21 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Signals.Comments (count).chromium.png|312| Changed|
|Signals.Emailed.chromium.png|312| Changed|
|Signals.Blocked.chromium.png|312| Changed|
|Signals.Mention.chromium.png|312| Changed|
|Signals.Awaiting approval.chromium.png|312| Changed|
|Signals.Comments.chromium.png|312| Changed|
|Signals.Follow.chromium.png|312| Changed|
|Signals.Malware detected.chromium.png|312| Changed|
|Signals.Missing metadata.chromium.png|312| Changed|
|Signals.New (positioning).chromium.png|312| Changed|
|Signals.New.chromium.png|312| Changed|
|Signals.Read-only.chromium.png|312| Changed|
|Signals.NotFollow.chromium.png|312| Changed|
|Signals.Someone checked out.chromium.png|312| Changed|
|Signals.Shared.chromium.png|312| Changed|
|Signals.Record.chromium.png|312| Changed|
|Signals.Trending.chromium.png|312| Changed|
|Signals.Warning.chromium.png|312| Changed|
|Signals.Unseen reply.chromium.png|312| Changed|
|Signals.Unseen edit.chromium.png|312| Changed|
|Signals.You checked out.chromium.png|312| Changed|

</details>


  <details><summary><b>Tile</b> 5 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Tile.Document tile with fit landscape image.chromium.png|312| Changed|
|Tile.Document tile with fit portrait image.chromium.png|312| Changed|
|Tile.Document tile with icon.chromium.png|312| Changed|
|Tile.Document tile with icon-sized image.chromium.png|312| Changed|
|Tile.Tile with no content and long text.chromium.png|312| Changed|

</details>


  <details><summary><b>react-charting-LineChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-LineChart.Gaps.chromium.png|1| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>